app/views/shared/vacancy/_jobseeker_view.html.slim

```diff
@@ -17,13 +17,8 @@
     .govuk-grid-row
       - if JobseekerApplicationsFeature.enabled? && @vacancy.apply_through_teaching_vacancies == "yes" && @vacancy.expires_at.future?
         .govuk-grid-column-one-third
-          - case job_application&.status
-          - when "submitted"
-            = render BannerButtonComponent.new text: t("jobseekers.job_applications.banner_links.submitted"),
-              href: jobseekers_job_application_path(job_application),
-              icon: "green-tick"
-          - when "draft"
-            = render BannerButtonComponent.new text: t("jobseekers.job_applications.banner_links.draft"),
+          - if defined?(job_application) && job_application.present?
+            = render BannerButtonComponent.new text: t("jobseekers.job_applications.banner_links.#{job_application.status}"),
```

Comment:
A job application can be more statuses than `submitted` and `draft` so we should think about what we say for each. I reckon that we probably want to distinguish between `draft` and `not_draft` rather than being that granular though